app/representation.py

```diff
@@ -3,21 +3,52 @@ def venueRecord(biz, **details):
     return {
       "version": 1.0,
       "id": biz.id,
+
+      "name": "A Temporary Name",
+      "summary": biz.snippet_text,
       "coordinates": {
         "lat": biz.location.coordinate.latitude,
-        "lon": biz.location.coordinate.longitude
+        "long": biz.location.coordinate.longitude
       },
-      "images": [],
-      "address": biz.location.display_address,
 
-      "pullQuote": biz.snippet_text,
+      "categories": ["Temporary Hotel"],
+      "url": "http://mozilla.org",
+
+      "address": biz.location.display_address,
 
       "providers": {
         "yelp": {
           "rating": biz.rating,
-          "reviewCount": biz.review_count,
-          "ratingMax": 5,
```

Comment:
On the client, we're using a scale of 0-5 with float values (which we round to 1/2s).

We could normalize on the client or the server – I figured we'd just do it on the server and reduce complexity on the client.